pom.xml

```diff
@@ -102,7 +102,7 @@
                     <configuration>
                         <testSources>
                             <testSource>
-                                <directory>${project.basedir}/src/test/groovy</directory>
+                                <directory>${project.basedir}/src/test</directory>
                                 <includes>
                                     <include>**/*.groovy</include>
                                 </includes>
```

Comment:
Instead of "match all" rule we can add "match UT" and we will cover all scenarios.

Reply:
we need to specify all tests in gmavenplus-plugin, even if tests are separated